ynput/OpenPype
# Nuke: creator farm attributes from deadline submit plugin settings

## Brief description
Defaults in farm attributes are sourced from settings.

## Description
Settings for deadline nuke submitter are now used during nuke render and prerender creator plugins. 

## Testing notes:
1. Open Studio Project settings at your testing project at `project_settings/deadline/publish/NukeSubmitDeadline`
2. set following values to something like this (img) notice attributes moved to right panel
![image](https://user-images.githubusercontent.com/40640033/221585770-f6b686da-84d4-4356-879e-6a701ae8a2ef.png)

3. Open Nuke in testing project to some shot and create new render write node. 
4. Notice that created node is having defined values as it were in deadline plugin settings
